pkg/gui/main_panel.go

```diff
@@ -22,9 +22,21 @@ func (gui *Gui) scrollDownMain(g *gocui.Gui, v *gocui.View) error {
 		_, sy := mainView.Size()
 		y += sy
 	}
+
+	_, sizeY := mainView.Size()
+	scrollCount := gui.Config.UserConfig.Gui.ScrollHeight
+	totalLines := mainView.LinesHeight()
```

Comment:
Using LinesHeight causes an issue when we have wrapping turned on with:
```
gui:
  wrapMainPanel: true
```
The issue is that you can press down and actually end up scrolling up because of the mismatch between the total original of lines and the number of lines when wrapping is done.

I've just pushed a commit to the gocui master branch which adds a `ViewLinesHeight()` function that you can use instead and that should fix that bug :)

If you need help bumping the gocui version let me know :)

Reply:
Hi there, do share with me what should i do to update the version. Thank you.

Reply:
Done ! I got it updated :)